src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-capabilities.mdx

```diff
@@ -0,0 +1,275 @@
+---
+title: User management capabilities 
+tags:
+  - Accounts
+  - New Relic One user management
+metaDescription: An explanation of New Relic user capabilities and what they govern. 
+redirects:
+---
+
+import newRelicUserCapabilitiesUi from 'images/user-management-capabilities.png'
+
+A role, whether one of our [pre-built roles](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#roles) or a custom role, is a set of capabilities. You can think of a "capability" as a granular thing a New Relic user can do. Examples of capabilities are: the capability to view APM app settings, or modify alert conditions, or manage data retention settings. 
+
+To learn what capabilities a role has, go to the user management UI and view a specific role. The location of this UI depends on what [user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models) a user is on: 
+
+* Newer user model: from [one.newrelic.com](https://one.newrelic.com), click **[account dropdown](/docs/new-relic-solutions/get-started/glossary/#account-dropdown) &gt; Administration &gt; Access management &gt; Roles**.  
+* Original user model: from [one.newrelic.com](https://one.newrelic.com), click **[account dropdown](/docs/new-relic-solutions/get-started/glossary/#account-dropdown) &gt; Account settings &gt; Users and roles**.  
+
+## How capabilities relate to roles [#capabilities-roles]
+
+Groupings of capabilities are aggregated into roles. We have pre-built roles for common use cases, and you can build your own custom roles. 
+
+Our pre-built roles work differently depending on which of our [user models](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models) you're on: 
+
+* Our newer user model: [learn about roles](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts). 
+* Our original user model: [learn about roles](/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model). 
+
+## Some important tips for using capabilities [#important-points]
+
+Some important points for understanding and using capabilities: 
+
+* **Capabilities can change.** The capabilities we expose in the UI are subject to change and reorganization. The capabilities in this doc were last updated June, 2022.     
+* **Some capabilities have overlapping functionality.** That is why selecting some capability checkboxes will automatically check or uncheck other boxes.
+* **Not all capabilities are exposed.** Some roles have hidden capabilities that are not visible in the UI and not available for addition to a custom role. For example, the **Organization manager** role has capabilities related to managing and configuring organization-level settings, but those are not available for addition to a custom role. 
```

Comment:
Is there a list of roles that are not "exposed" in addition to organization manager? Is it in the UI that you can reference? (If you don't think it's helpful or you think it might be redundant, disregard.)